common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -151,17 +253,22 @@ div.problem {
     }
   }
 
+  // known classes using this div: .indicator_container, moved to section above
   div {
+
+    // TO-DO: Styling used by advanced capa problem types. Should be synced up to use .status class
     p {
       &.answer {
         margin-top: -2px;
       }
+
       &.status {
-        margin: 8px 0 0 $baseline/2;
+        @include margin(8px, 0, 0, $baseline/2);
```

Comment:
Can you wrap the last value here in another set of quotes? I., `(8px, 0, 0, ($baseline/2))`